codemon/CodemonHelp.py

```diff
@@ -4,7 +4,7 @@ def showHelp():
   print(colored.green("           ---CODEMON---          "))
   print("A CLI tool to ace competitive programming contests")
   print(colored.cyan("\nCOMMANDS: \n"))
-  print("codemon - - - - - - - - - - - - - - -  displays this message")
+  print("codemon - - - - - - - - - - - - - - -  shows help")
   print("codemon --help - - - - - - - - - - - - shows help")

```

Comment:
This change was done in a recently merged PR, could you replace the string related to `codemon --help` to "displays this message" ?